### DISCUSSION TITLE
Does bb support M1 macbook?

### DISCUSSION THREAD

**Discussion by temco** [babashka/babashka#1068] [Q&A]
Can I download the macos_amd64 version as the alternative, thanks!

**Comment by grzm** [comment]
The babashka macos_amd64  build runs just fine under rosetta on with an M1 processor.

**Comment by borkdude** [comment]
@temco Babashka is compiled with GraalVM and currently GraalVM doesn't have a compiler for M1. You can however just use the macos_amd64 binary like @grzm suggested and run it with Rosetta2.
